README.md

```diff
@@ -33,3 +33,22 @@ sudo cp nrf802154_sniffer/nrf802154_sniffer.py /usr/lib/x86_64-linux-gnu/wiresha
 3. Select the 802.15.4 channel.
 4. Select the serial port associated with the board that you flashed the firmware on.
 5. Click 'Start'.
+
+## Custom Wireshark dissector
+Custom wireshark dissector can be used to obtain additional informations from sniffer. Channel, RSSI and LQI can be displayed for every packet.
+
+### Install lua dissector
+Copy the provided script to the approperiate directory:
+```
+sudo cp nrf802154_sniffer/nrf802154_sniffer.lua /usr/lib/x86_64-linux-gnu/wireshark/plugins/
+```
+### Modify extcap script
+Modify `nrf802154_sniffer.py`:
```

Comment:
Could you add a command example? Like `sudo nano /usr/lib/x86_64-linux-gnu/wireshark/extcap/nrf802154_sniffer.py`?

Reply:
That's the idea for the future. I think we should merge this PR as it is and improve it in the future.